### DISCUSSION TITLE
How to resolve the '[DOM] Input elements should have autocomplete attributes' warning in Chrome?

### DISCUSSION THREAD

**Discussion by QLesson** [community/community#141500] [Programming Help]
### Body

How to resolve the '[DOM] Input elements should have autocomplete attributes' warning in Chrome?

Description:
I'm working on a web application, and I encountered a browser warning in Chrome's developer tools that says:

[DOM] Input elements should have autocomplete attributes (suggested: 'current-password')

This warning appears when I have a form with a password input field. Below is the form I'm using:

```js
<form method="POST" action="/login">
    <input type="text" name="username" placeholder="Username">
    <input type="password" name="password" placeholder="Password">
    <button type="submit">Login</button>
</form>
```

I'm unsure how to properly address this warning and what autocomplete attributes are expected for the input elements.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by aldiipratama** [comment]
Add autocomplete to Input Element To remove the warning, add the appropriate autocomplete attribute to the input element. In the case of password input, the recommended attribute is autocomplete=“”.

Example:
```
<input type=“text” name=“username” autocomplete=“username”>
<input type=“password” name=“password” autocomplete=“current-password”>
```

**Comment by ebndev** [comment]
Please see [this comment](https://github.com/orgs/community/discussions/141501#discussioncomment-10952853) for reference to why this discussion was closed and locked. Thanks!
